applications/nrf_desktop/doc/ble_discovery.rst

```diff
@@ -29,7 +29,8 @@ Complete the following steps to configure the module:
 #. Set the :kconfig:`CONFIG_BT_GATT_CLIENT` Kconfig option to enable support for the GATT Client role.
 #. Set the :kconfig:`CONFIG_BT_GATT_DM` Kconfig option to enable the :ref:`gatt_dm_readme`.
    The :ref:`gatt_dm_readme` is used by the ``ble_discovery`` application module.
-#. Define the module configuration in the :file:`ble_discovery_def.h` file, located in the board-specific directory in the application configuration directory.
+#. Define the module configuration in the configuration file specified by the :kconfig:`CONFIG_DESKTOP_BLE_DISCOVERY_DEF_PATH option.
+   The configuration file should be placed in the board-specific directory in the application configuration directory.
    You must define the following parameters for every nRF Desktop peripheral that connects with the given nRF Desktop central:
 
    * Parameters common for all the peripherals:
```

Comment:
```suggestion
#. Make sure to specify the configuration file in the :kconfig:`CONFIG_DESKTOP_BLE_DISCOVERY_DEF_PATH` option and that the file is located in the board-specific directory in the application configuration directory.
#. Define the module configuration in the configuration file by defining the following parameters for every nRF Desktop peripheral that connects with the given nRF Desktop central:

   * Parameters common for all the peripherals:
```
Would the user be able to find this Kconfig option when going through these steps?

Reply:
I don't see why not, its well described, step by step. Suggestion applied.